onelogin_aws_cli/configuration.py

```diff
@@ -7,32 +7,49 @@
 class ConfigurationFile(configparser.ConfigParser):
     """Represents a configuration ini file on disk"""
 
+    DEFAULTS = dict(save_password=False)
+
     def __init__(self, config_file=None):
         super().__init__(default_section='defaults')
 
-        self['defaults'] = dict(save_password=False)
+        self[self.default_section] = self.DEFAULTS
 
         self.file = config_file
 
         if self.file is not None:
             self.load()
 
+    @property
+    def has_defaults(self) -> bool:
+        """True if the defaults section has settings beyond our defaults"""
+        return len(self.defaults()) > len(self.DEFAULTS)
+
     @property
     def is_initialised(self) -> bool:
-        """True if there is at least one section"""
+        """True if there is at least one section or a defaults section"""
         return len(
             self.sections()
-        ) > 0
+        ) > 0 or self.has_defaults
 
     def load(self):
         self.read_file(self.file)
-
-    def initialise(self, config_name='default'):
+        # For backwards compatibility, we check if they have a default
+        # section instead of a defaults section and do a little switcheroo as
+        # needed.
+        if self.has_section('default') and not self.has_defaults:
```

Comment:
This might get a bit confusing.
I saw:
 - the `default` section as being the config which is used when no arguments are provided to the cli.
 - the `defaults` section as being an "abstract" config which will fill in the gaps in the other configs.

If we say `defaults` and `default` are the same, which config is picked when no cli args are set? I would like to be be able to set some sort of default config, as for the most time I would be setting the same option again and again on the cli (`onelogin-aws-login -C my_config`)